adapters/lora-vertx-base/src/main/java/org/eclipse/hono/adapter/lora/LoraProtocolAdapter.java

```diff
@@ -428,7 +428,8 @@ private void handleCommand(final CommandContext commandContext) {
         final Command command = commandContext.getCommand();
         final Promise<Void> sendPromise = Promise.promise();
         final Buffer payload = Optional.ofNullable(command.getPayload()).orElseGet(Buffer::buffer);
-        final LoraCommand loraCommand = loraProvider.getCommand(commandEndpoint, command.getDeviceId(), payload);
+        final String subject = Optional.ofNullable(command.getName()).orElseGet(String::new);
```

Comment:
Command.getName() never returns `null`

Reply:
changed it to directly put it in the string

Reply:
changed it to directly put it in the string